src/lib/autocomplete/autocomplete-trigger.ts

```diff
@@ -121,6 +121,9 @@ export class MdAutocompleteTrigger implements ControlValueAccessor, OnDestroy {
   openPanel(): void {
     if (!this._overlayRef) {
       this._createOverlay();
+    } else {
+      /** Gets width of host incase this has changed, and updated the Overlay reference */
```

Comment:
This isn't very understandable. Something along the lines of "Update the panel width, in case the host has changed" might be better.